app/views/content/partials/subject-pages/_adviser-promo.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.with_left_section(classes: %w[tta-background]) %>
+  <% promo.with_right_section(heading: "Get free one-to-one support") do %>
+    <% subject = @front_matter["subject"] %>
```

Comment:
Not sure I'd agree with you @martyn-w - this promo will be used on other subject pages including `art and design` where the article will need to be different?

Reply:
sorry, to be clear I mean, rather than:

```
subject = @front_matter["subject"]
article = @front_matter["article"] || "a"
...
....can help you to become <%= article %> <%= subject %> teacher...
...
```

you could simplify with:

```
....can help you to become <%= @front_matter["article"] || "a" %> <%= @front_matter["subject"] %> teacher...
```

there isn't much point in storing these values in variables if they are only used once

Reply:
This would be a good place for a `indefinite_article(word)` helper that returns `an` if the word starts with a vowel else returns `a`

Reply:
Thanks all! Have removed the variables and `indefinite_article` gem works a treat! 👍